runtime/nvqir/cutensornet/mps_simulation_state.h

```diff
@@ -96,6 +96,11 @@ class MPSSimulationState : public cudaq::SimulationState {
   std::unique_ptr<TensorNetState> state;
   std::vector<MPSTensor> m_mpsTensors;
   ScratchDeviceMem m_scratchPad;
+  // Max number of qubits whereby the tensor network state should be contracted
+  // and cached into a state vector.
+  // This speeds up sequential state amplitude accessors for small states.
+  static inline constexpr std::size_t g_maxQubitsForStateContraction = 30;
```

Comment:
```suggestion
  static constexpr std::size_t g_maxQubitsForStateContraction = 30;
```
inline isn't needed here.